### PR TITLE
Concept publisher checking on good-to-go

### PR DESCRIPTION
**Pull request by kovacshuni**

### CODE REVIEW

File: concept-publisher@.service [hunk 1]
```diff
@@ -14,11 +14,13 @@ ExecStartPre=-/bin/bash -c 'docker kill "$(docker ps -q --filter=name=%p-%i_)" >
 ExecStartPre=-/bin/bash -c 'docker rm "$(docker ps -q --filter=name=%p-%i_)" > /dev/null 2>&1'
 ExecStartPre=/bin/bash -c 'docker history coco/concept-publisher:$DOCKER_APP_VERSION > /dev/null 2>&1 || docker pull coco/concept-publisher:$DOCKER_APP_VERSION'
 ExecStart=/bin/bash -c '\
+  TRANSFORMER-GTG-RETRIES=$(etcdctl get /ft/config/concept-publisher/gtg-retries) || TRANSFORMER-GTG-RETRIES=30;
 	docker run --rm --name %p-%i_$(uuidgen) -p 8080 \
 	--memory="1024m" \
 	--env "PROXY_ADDRESS=http://%H:8080/__kafka-rest-proxy" \
 	--env "TOPIC=Concept" \
 	--env "CLUSTER_ROUTER_ADDRESS=http://%H:8080/" \
+  --env "TRANSFORMER-GTG-RETRIES=$TRANSFORMER-GTG-RETRIES" \
```
Comment (Nesze) [inline concept-publisher@.service]: formatting

Comment (kovacshuni) [inline concept-publisher@.service]: this file has tabs, the lines i added are with spaces, i don't know what to do :robot_face:
screen shot 2017-01-11 at 11 47 00

Comment (Nesze) [inline concept-publisher@.service]: use tabs? :) just edit it here online and benefit from the preview feature

Comment (kovacshuni) [inline concept-publisher@.service]: done :)

**Pull request merged**